packages/liveblocks-core/src/__tests__/auth-manager.test.ts

```diff
@@ -46,34 +46,34 @@ describe("auth-manager - secret auth", () => {
     "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjE2NjQ1NjY0MTAsImV4cCI6MTY2NDU3MDAxMCwicGlkIjoiNjA1YTRmZDMxYTM2ZDVlYTdhMmUwOGYxIiwidWlkIjoidXNlcjEiLCJrIjoiaWQifQ.MhjHg2udkBivX7cW8Q1jmgc4DOZ1YnMoUnP61O32JLVJlIsc0zmHWA__DItsO3vBbRS8doG98cOzSE2qQ-5rKoX2l19k5Mr7gk6M75u1kOAzppV_3YQAGeZ8PfsUUPUBGOF5O6msLnha-HcAywvBuoUmcqP0CF_xhBBx0CLbFeuaWVJqndPKe8LJk9EYcB29HEwFaIzrOSarU1iLxRhsa8FCB910GTDcaApaUTPM9ZRadmf33ypSn3c6by0BWI54vx4O2p-hFsmJ71R38ifRRVq3ETXn78ftwbu1pp6hMTqyYn5YLlnZPPM-JAck_OsarGvE9cxg_Z3Y8bMTOlA5E";
 
   const server = setupServer(
-    rest.post("/mocked-api/legacy-auth", (_req, res, ctx) => {
-      return res(
-        ctx.json({ token: legacyTokens[requestCount++ % legacyTokens.length] })
-      );
+    http.post("/mocked-api/legacy-auth", () => {
```

Comment:
The changes are rather minimal but they bring your tests much closer to the platform. If you are curious, [read about the 2.0 release](https://mswjs.io/blog/introducing-msw-2.0).